llvm/test/CodeGen/AArch64/alias_mask.ll

```diff
@@ -0,0 +1,1556 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=aarch64 -mattr=+sve2 %s -o - | FileCheck %s --check-prefix=CHECK-SVE
+; RUN: llc -mtriple=aarch64 %s -o - | FileCheck %s --check-prefix=CHECK-NOSVE
```

Comment:
I'm not sure how useful these `NOSVE` RUN lines are, because they test the same expand code for every variant. I think testing that testing that code once would be enough. Maybe you can move _one_ such case (e.g. one with legal types) to a separate file and then have a RUN line with NOSVE, and then remove the NOSVE RUN-line from this test (and also for alias_mask_scalablel.ll). That would reduce the test files a bit.